src/main/java/tuteez/logic/commands/EditCommand.java

```diff
@@ -97,24 +97,18 @@ public CommandResult execute(Model model) throws CommandException {
                     continue;
                 }
 
-                if (checkForClashingLesson(newLesson)) {
+                if (model.isClashingWithExistingLesson(newLesson)) {
                     throw new CommandException(MESSAGE_DUPLICATE_LESSON);
                     //return new CommandResult(MESSAGE_DUPLICATE_LESSON, false, false, false);
```

Comment:
Perhaps this comment can be removed if it's not used for neater code

Reply:
Alright!! The comment is actually there because I was considering an implementation where a warning message pops up if there conflicting students but `tuteez` actually allows duplicate lessons if user confirms. In that case I cannot just throw `CommandException`. I will remove the command for now thought!! :)